doc/GIT_USAGE.md

```diff
@@ -19,23 +19,23 @@
 ## Branching Model
 
 - The DMOD repo uses a branching model based on [Gitflow](https://nvie.com/posts/a-successful-git-branching-model/) and has two primary long-term branch: 
-  - **master**: the main branch pointing to the current, official version
-  - **dev**: an integration branch containing the latest completed development work intended for the next released version
-- Most interaction with DMOD repo is done via pull requests (PRs) to the `dev` branch
-  - Independent branches for features or bug fixes are created off `dev` to contain development work that is in progress
-  - These branches are reviewed and their changes integrated back into `dev` once complete via PRs
+  - **master**: the main development and integration branch containing the latest completed development work intended for the next released version
+  - **production**: the branch representing the latest code verified as production-ready and pointing to the most recently release, official version
+- Most interaction with DMOD repo is done via pull requests (PRs) to the `master` branch
+  - Independent branches for features or bug fixes are created off `master` to contain development work that is in progress
+  - These branches are reviewed and their changes integrated back into `master` once complete via PRs
   - Typically feature/fix branches exist in personal clones and personal Github forks, but not in the official OWP repo
 - Release branches (e.g., `release-X` for pending version `X`) will periodically be created to finalize the next new version when it is time for it to be released
   - These are managed by the core OWP contributors team
```

Comment:
Can you emphasize that 99% of people interacting with this repo will _not_ be involved in release or production branches unless they are writing fixes for production releases?

Reply:
There's a sign of my own confusion since they won't really be for fixes, but release candidates. Is there any way you can specify or elaborate on that?

Reply:
> Can you emphasize that 99% of people interacting with this repo will not be involved in release or production branches unless they are writing fixes for production releases?

I am going to add this immediately under the title heading at the top of the file (and also add a bunch of the practical stuff to `CONTRIBUTING.md`):

> Note that this document goes into detail on the Git strategy and branching model for DMOD.  It is here for openness and transparency, but many (if not most) users will not need to be concerned with this level of detail.  For information geared toward day-to-day development contributions and Git, see the CONTRIBUTING doc.

Does that seem sufficient, or should there also be something here to that effect?

Reply:
yeah, it's great